src/app/modules/users/add-user/add-user.component.ts

```diff
@@ -0,0 +1,292 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { DynamicFormComponent } from '../../admin-shared';
+import { FieldConfig } from "../../admin-shared/field.interface";
+import { FormBuilder, FormGroup, FormControl, Validators, FormArray } from '@angular/forms';
+import { UsersService } from '../../admin-core';
+import { MatDialog, MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+
+
+@Component({
+  selector: 'app-add-user',
+  templateUrl: './add-user.component.html',
+  styleUrls: ['./add-user.component.scss']
+})
+export class AddUserComponent implements OnInit {
+  @ViewChild(DynamicFormComponent) form: DynamicFormComponent;
+  questions: any[];
+  field: FieldConfig;
+  group: FormGroup;
+  // regConfig: any[];
+  formdata: any;
+  fieldsbackend: any;
+  loading: boolean = false;
+  regConfig: FieldConfig[] = [
```

Comment:
this variable should be populated  from backend api

Reply:
yes, the fields will populate from backend, currently some regular expression in invalid, and it is collapsing entire form to build